inputs/AccountSettings.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Text, View, TextInput, TouchableOpacity, ScrollView, SafeAreaView } from 'react-native';
+import styles from '../styles';
+
+export default function UserSettings() {
+    const [newPassword, setNewPassword] = useState('');
```

Comment:
This is adding extra logic separately when we could combine everything in one object state, as we did in the inputs lesson, to maintain readability